thesis-156/run-thesis-156.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+docker network create -d bridge --subnet=172.20.0.0/16 --gateway=172.20.0.1 thesis-156
+
+docker build --file THESIS-155/Dockerfile -t thesis-155:latest .
+
+docker run -d --network=thesis-156 --ip=172.20.0.5 --name=db-server thesis-155:latest
```

Comment:
รัน ตบบรรทัดแล้ว ปลั๊ก Volume กันเถอะ จะได้เก็บ db ใน volume ไปไว้ทำอย่างอื่นต่อได้

```suggestion
docker run --rm -d \
    --network=thesis-156 \
    --ip=172.20.0.5 \
    --name=db-server \
    -v vol-thesis-156:/var/lib/postgresql/data \
    thesis-155:latest
```